### ISSUE TITLE
"Add NuGet Package " text is not having 4.5:1 color contrast ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#280]
### Steps to Reproduce

1. Launch Xamarin Work application. It will open "Create a new workbook" window.
2. Navigate to "+" (Add Nuget Package) symbol and activate it.
3. Take a screenshot and verify the colour contrast ratio.

### Expected Behavior
Colour contrast ratio should be minimum 4.5:1

### Actual Behavior
Colour contrast ratio is less than 4.5:1
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

### Log File

![add nuget package](https://user-images.githubusercontent.com/22956130/39864933-c84eb7cc-543a-11e8-8c35-27c46e46dd48.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by abock** [comment]
This is a stock macOS control. We do not influence or control the rendering of this control. The system-wide "Graphite" appearance may exhibit 4.5:1 contrast. Regardless, this is not something we can or should address in our code as it comes from macOS itself.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
